lib/ansible/modules/command.py

```diff
@@ -24,6 +24,8 @@
        arguments, pass parameters using the C(args) L(task keyword,../reference_appendices/playbooks_keywords.html#task)
        or use C(cmd) parameter.
      - Either a free form command or C(cmd) parameter is required, see the examples.
+     - Free-form arguments can create unexpected syntax errors (such as Jinja2 parsing errors from apostrophes 
+       within a comment). Use the I(cmd) argument instead.
```

Comment:
not sure I follow you on this. Do you mean the `raw` listed in the attributes table should have this snippet of information instead of here? aka https://github.com/ansible/ansible/blob/9cd05708145ebff825fabd62edfe3cfcae3e25e7/lib/ansible/modules/command.py#L42

Reply:
yes, that or the _raw_params option itself